### DISCUSSION TITLE
Horizontal mouse wheel scroll NOT WORKING

### DISCUSSION THREAD

**Discussion by judeabufarha** [pmndrs/react-spring#1750] [Support]
How do you get the mouse scroll to work with the wheel for horizontal scroll parallax layers. At the moment it only works with the default windows scroll bar at the bottom of the screen. This seems to automatically work with the mouse for vertical scroll parallax websites but not a horizontal scroll parallax sites.  All help is appreciated :) 

Thanks!!

**Comment by JHunnicutt** [comment]
I too have this question. I tried implementing a number of things, but they don't seem to work all that well with React Spring's Parallax component.
